src/authentication/register.js

```diff
@@ -0,0 +1,31 @@
+const jwt = require( 'jsonwebtoken' )
+const crypto = require( 'crypto' )
+const User = require( '../../models/users' )
+const { generateToken, userInfo, tokenInfo } = require( './authmethods' )
+
+const ERROR_MESSAGE = 'Cannot authenticate request'
+
+exports.register = ( request, response, next ) => {
+  const { email, password } = request.body
+  if (!email) {
```

Comment:
Spaces between parens and args, please omit spaces between keywords and opening parens.